README.md

```diff
@@ -165,6 +165,7 @@ All pull requests are verified on the following devices (ordered by Android vers
 * [Huawei Y560-L01](http://www.gsmarena.com/huawei_y560-7829.php) - Android 5.1.1
 * [Sony Xperia Z3 D6603](http://www.gsmarena.com/sony_xperia_z3-6539.php) - Android 6.0.1
 * [Samsung Galaxy S7 SM-G930F](https://www.gsmarena.com/samsung_galaxy_s7-7821.php) - Android 7.0.0
+* [Motorola Moto G4 Play](https://www.gsmarena.com/motorola_moto_g4_play-8104.php) - Android 7.1.1
 * [LG Nexus 5X](https://www.gsmarena.com/lg_nexus_5x-7556.php) - Android 8.1
 * [Xiaomi Redmi 7](https://www.gsmarena.com/xiaomi_redmi_7-9498.php) - Android 9.0
 * [Google Pixel 3a](https://www.gsmarena.com/google_pixel_3a-9408.php) - Android 10.0
```

Comment:
It would be good to update the section below as well (`Our regular code contributors use these devices (ordered by Android version):`)
For example Samsung Galaxy S4 GT-I9506 was mine and I no longer use it.
I don't know who used other devices @lognaturel @shobhitagarwal1612 @seadowg anybody was the owner of any of those devices?

Reply:
Nope not me!

Reply:
I have the Samsung Galaxy Tab SM-T285 - Android 5.1.1 and Motorola G 5th Gen XT1671 - Android 7.0 and still use both! I also have access to a Pixel 3A with Android 11 but use it less frequently and the Pixel 3a is already mentioned above.

Reply:
> I also have access to a Pixel 3A with Android 11 

@lognaturel, do you want this device to be included in the list, even if it is used less frequently?

> For example Samsung Galaxy S4 GT-I9506 was mine and I no longer use it.

@grzesiek2010, mentioned device was removed.

Reply:
Do we know who is the owner of the Xiaomi Redmi Note 4?

Reply:
I think it would be a good idea to tag people that use those devices like:
[Samsung Galaxy Tab SM-T285](http://www.gsmarena.com/samsung_galaxy_tab_a_7_0_(2016)-7880.php) - Android 5.1.1 [@lognaturel](https://github.com/lognaturel)

That Xiaomi devices was added by @shobhitagarwal1612 https://github.com/getodk/collect/pull/1847/files so I think we can remove it.

Reply:
Let's not add the other Pixel 3A. Agreed with removing Xiaomi devices and tagging device owners sounds good.